content/README.md

```diff
@@ -0,0 +1,73 @@
+# Apache Beam (incubating) website
```

Comment:
A little puzzled why this file showed up in the `content` directory. Accidental / intentional?